docs/tempo/website/api_docs/_index.md

```diff
@@ -151,7 +151,7 @@ The URL query parameters support the following values:
 - `start = (unix epoch seconds)`
   Optional.  Along with `end` define a time range from which traces should be returned. 
 - `end = (unix epoch seconds)`
-  Optional.  Along with `start` define a time range from which traces should be returned. Providing both `start` and `end` will change the way that Tempo searches. If the parameters are not provided then Tempo will search the recent trace data stored in the ingesters. If the parameters are provided it will search the backend as well.
+  Optional.  Along with `start` define a time range from which traces should be returned. Providing both `start` and `end` will change the way that Tempo searches. If the parameters are not provided then Tempo will search the recent trace data stored in the \s. If the parameters are provided it will search the backend as well.
```

Comment:
```suggestion
 Optional.  Along with `start`, define a time range from which traces should be returned. Providing both `start` and `end` will change the way that Tempo searches. 
 If the parameters are not provided, then Tempo will search the recent trace data stored in the ingesters. If the parameters are provided, it will search the backend as well.
```